src/css/custom.css

```diff
@@ -14,6 +14,7 @@
   --ifm-list-item-margin: 0;
   --ifm-code-font-size: 95%;
   --ifm-z-index-fixed: 1;
+  --ifm-leading: .8rem;
```

Comment:
Just curious how you landed on this number. Wondering if it's worth just keeping it as 1rem so it lines up with the other standard spacing? Either way is fine, they both work. I am more curious than anything else